ext/storage_sources/azure_store/azure_storage_source.cpp

```diff
@@ -0,0 +1,275 @@
+
+#include <wiredtiger.h>
+#include <wiredtiger_ext.h>
+#include <list>
+
+#include "azure_connection.h"
+#include "wt_internal.h"
+
+struct azure_store {
+    WT_STORAGE_SOURCE store;
+    std::list<azure_file_system *> azure_fs;
```

Comment:
Is there a good reason to prefer a list over a vector here? (Same for below.)

Reply:
thanks for the suggestion, changed to use vector